docs/pages/debugging/tools.mdx

```diff
@@ -7,17 +7,110 @@ sidebar_title: Tools
+  src="/static/images/debugging/inspector-breakpoint-console.png"
+/>
+
+### Inspecting network requests
+
+The Network tab gives you insights in the network requests made by your app. You can inspect each individual requests and responses by clicking on them. This includes `fetch` requests, external loaded media, and in some cases even requests made by native modules.
+
+This feature is only available when using the dev client, it is not available when using Expo Go.
+
+<ImageSpotlight
+  alt="Gain insights in the network requests from your app"
+  src="/static/images/debugging/inspector-network-post.png"
+/>
+
+> **info** See the [Inspecting network traffic](#inspecting-network-traffic) for alternative ways to inspect network requests.
+
+## Debugging with VS Code
+
+VS Code is a popular code editor, which has a built-in debugger. This debugger uses the same system as the Chrome DevTools, the inspector protocol.
+
+You can use this debugger with the [Expo Tools](https://github.com/expo/vscode-expo#readme) VS Code extension. This debugger allows you to set breakpoints, inspect variables, and even execute code through the debug console.
+
+<ImageSpotlight
+  alt="Debug your code while you write it"
+  src="/static/images/debugging/vscode-expo.png"
+/>
+
+To start debugging, connect your app and run the "Expo: Debug ..." VS Code command. This will attach VS Code to your running app.
+
+## Debugging with React DevTools
+
+React DevTools is a great way to get a look at each of your components' props and state. You can open the React DevTools by pressing <kbd>Shift</kbd> + <kbd>m</kbd> in the terminal where Expo was started. Once you have opened the React DevTools, it will appear as below:
+
+<ImageSpotlight
+  alt="Inspect your components using the React DevTools"
+  src="/static/images/debugging/react-devtools.png"
+/>
+
+React DevTools can also be paired with the [element inspector](#showhide-element-inspector), allowing you to inspect props, state, and instance properties by tapping components inside your app.
+
+## Developer menu
+
+This menu gives you access to several functions which are useful for debugging and is built into the Expo Go or dev client apps. If you are using the emulator, simulator, or have your device connected via USB, you can open this menu by pressing <kbd>m</kbd> in the terminal where Expo was started.
+
+<Collapsible summary="Or use any of the alternative options">
+
+- Android device (without USB): Shake the device verically.
+- Android Emulator or device (with USB):
+  - Press <kbd>Cmd ⌘</kbd> + <kbd>m</kbd> or <kbd>Ctrl</kbd> + <kbd>m</kbd>
+  - Run `adb shell input keyevent 82`
+
+- iOS device (without USB):
+  - Shake the device.
+  - Touch 3 fingers to the screen.
+- iOS simulator or device (with USB):
+  - Press <kbd>Ctrl</kbd> + <kbd>Cmd ⌘</kbd> + <kbd>z</kbd> or <kbd>Cmd ⌘</kbd> + <kbd>d</kbd>
+
+</Collapsible>
 
 Once you have opened the Developer menu, it will appear as below:
```

Comment:
Unrelated to this PR: the screenshot below this line is outdated.

@byCedric, happy to update this in this or a follow up PR. 

Here is the latest screenshot.

<img width="509" alt="CleanShot 2023-06-12 at 22 43 48@2x" src="https://github.com/expo/expo/assets/10234615/720d9135-13cf-462b-b824-af9bf917d361">